streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1354,14 +1346,12 @@ public void shouldHandleTimeoutExceptionInTransitRestoredTaskToRunning() {
         final TaskManager taskManager = setUpTransitionToRunningOfRestoredTask(task, tasks);
```

Comment:
don't we want to do a `taskManager.setMainConsumer(mockitoConsumer);` here? (unless setUpTransitionToRunningOfRestoredTask is already doing so)